wip/run-mysql-driver.php

```diff
@@ -0,0 +1,656 @@
+<?php // phpcs:disable WordPress.Files.FileName.InvalidClassFileName
```

Comment:
I'm assuming there's not much to review in this file yet so I'll skip over it. LMK if you wanted me to review this one as well.